webapp/packages/core-navigation-tree/src/NodesManager/NavTreeResource.ts

```diff
@@ -461,8 +461,8 @@ export class NavTreeResource extends CachedMapResource<string, string[], Record<
   }
 
   protected async loader(originalKey: ResourceKey<string>): Promise<Map<string, string[]>> {
-    const pageKey =
-      this.aliases.isAlias(originalKey, CachedResourceOffsetPageKey) || this.aliases.isAlias(originalKey, CachedResourceOffsetPageListKey);
+    const pageListKey = this.aliases.isAlias(originalKey, CachedResourceOffsetPageListKey);
+    const pageKey = this.aliases.isAlias(originalKey, CachedResourceOffsetPageKey) || pageListKey;
```

Comment:
I can see a copy-paste here and a little bit below for all resources with pagination.
Maybe we should add a helper here so it returns the correct `pageKey` and the `page`? So we don't miss this bug when we create a new feature with the exact pagination mechanism

Reply:
now you can use `getOffsetPageKeyInfo` helper